### PR TITLE
Irs super user cognito

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/persistence/dynamo/users/createUser.js [hunk 1]
```diff
@@ -69,6 +69,10 @@ exports.createUserRecords = async ({ applicationContext, user, userId }) => {
 
 exports.createUser = async ({ applicationContext, user }) => {
   let userId;
+  let userPoolId =
+    user.role === 'irsSuperuser'
```
Comment (rachaelparris) [inline shared/src/persistence/dynamo/users/createUser.js]: Could you pull this from the user role constants on the User entity?

File: web-api/serverless-cases.yml [hunk 2]
```diff
@@ -213,9 +213,8 @@ functions:
                 caseId: true
           cors: ui-${self:provider.stage}.ef-cms.${opt:domain}
           authorizer:
-            type: COGNITO_USER_POOLS
-            authorizerId:
-              Ref: ApiGatewayAuthorizer
+            arn: arn:aws:lambda:us-east-1:${opt:accountId}:function:cognito_authorizer_lambda_${self:provider.stage}
+            managedExternally: true
```
Comment (rachaelparris) [inline web-api/serverless-cases.yml]: Does this need to be added for more endpoints? Are you just doing this one for now for the POC?

**Review by matthopson — APPROVED** [review]

**Comment by matthopson** [comment]
Assuming we will be authorizing more endpoints when this is verified.

File: shared/src/persistence/dynamo/users/createUser.js (hunk 1 above)
Comment (matthopson) [reply shared/src/persistence/dynamo/users/createUser.js]: Agree. Good call

File: web-api/serverless-cases.yml (hunk 2 above)
Comment (codyseibert) [reply web-api/serverless-cases.yml]: yeah, I'm thinking all of them maybe

**Review by kkoskelin — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**